sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/JsonExpressionsSuite.scala

```diff
@@ -372,6 +372,58 @@ class JsonExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     )
   }
 
+  test("from_json - array") {
+    val schema = ArrayType(StructType(StructField("a", IntegerType) :: Nil))
+
+    // json array: `Array(Row(...), ...)`
+    val jsonData1 = """[{"a": 1}, {"a": 2}]"""
+    val expected =
+      InternalRow.fromSeq(1 :: Nil) ::
+      InternalRow.fromSeq(2 :: Nil) :: Nil
+    checkEvaluation(JsonToStruct(
+      schema, Map.empty, Literal(jsonData1), gmtId), expected)
```

Comment:
could you put input and expected output in different rows for readability please